Test/linters/constructorCaseWithoutParenthesis.dfy

```diff
@@ -0,0 +1,87 @@
+// RUN: %dafny /compile:0 "%s" > "%t"
+// RUN: %diff "%s.expect" "%t"
+
+module WithWarning {
+    datatype Color = Red | Green | ShadesOfGray(nat) 
+    
+    method MonochromaticMethod(c: Color) returns (x: bool) {
+          
+          return match c
+            case ShadesOfGray => true
+            case anythingElse => false;
+    }
+    method MonoMethod(c: Color) returns (x: bool) {
+              
+              return match c
+                case Green => true
+                case anythingElse => false;
+        }
+    
+    function method MonochromaticFunction(c: Color) : bool {
+        match c
+            case ShadesOfGray => true
+            case anythingElse => false
+    }
+    method MonochromaticMethodloop(c: Color) returns (x: bool)  {
+        var test := false;
+        while test
+        {
+            test := match c
+                case ShadesOfGray => true
+                case anythingElse => false;
+        }
+        return false; 
+    }
+   
+
+}
+
+module WithoutWarning {
+     datatype Color = Red | Green | ShadesOfGray(nat) | MixColors(nat, nat) 
```

Comment:
Could you make the indentation in this file consistent? This line starts with 5 indentation. Most of this file uses indentation steps of 4, but I'd rather see indentation steps of 2.